tests_new/integration_tests/client.py

```diff
@@ -0,0 +1,36 @@
+import requests
+import boto3
+import os
+from munch import DefaultMunch
+
+
+ENVNAME = os.getenv('ENVNAME', 'dev')
+
+
+class Client:
+    def __init__(self, username, password):
+        self.username = username
+        self.password = password
+        self.token = self._get_jwt_token()
+
+    def query(self, query: str):
+        graphql_endpoint = f'{os.getenv("API_ENDPOINT", False)}graphql/api'
+        headers = {'AccessKeyId': 'none', 'SecretKey': 'none', 'authorization': self.token}
+        r = requests.post(graphql_endpoint, json=query, headers=headers)
```

Comment:
nit: I know this is going to be replaced with SDK but does it make sense to use [GQL's client](https://github.com/graphql-python/gql)? Maybe it does have some retry mechanisms and might also provide better logging.

Reply:
I have been working a bit  with it and tried to use the httpx trasnport of the gql client, at the end I ended up writing the same code as the one we have in the CLI/SDK. I don't want to make it difficult to replace when the time comes, I just want to have the minimum functionality needed for the tests to work